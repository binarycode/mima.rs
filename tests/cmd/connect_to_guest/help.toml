args = 'help connect-to-guest'
status.code = 0
fs.sandbox = true
