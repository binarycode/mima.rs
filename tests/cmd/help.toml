args = 'help'
status.code = 0
fs.sandbox = true
