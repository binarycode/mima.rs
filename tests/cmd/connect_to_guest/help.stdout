Establish SSH connection to guest

Usage: mima connect-to-guest <GUEST_ID> [-- [ARGS]...]

Arguments:
  <GUEST_ID>  Guest ID
  [ARGS]...   Arguments to pass to SSH

Options:
  -h, --help  Print help
